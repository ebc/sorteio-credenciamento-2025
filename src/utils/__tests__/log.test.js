import {
  logParametros,
  logSorteio,
  logPareceristaAdicionado,
  logPareceristaNaoAdicionado
} from '../log';

it('loga corretamente parametros sorteio', () => {
  const pareceristas = [
    { 'Inscricao': '123', 'CNPJ': '48797752000151', 'RazaoSocial': 'Produtora xyz' }
  ];
  expect(
    logParametros('11/11/2018 11:11:11', pareceristas, 'semente1234')
  ).toMatchSnapshot();
});

it('loga corretamente sorteio', () => {
  const sorteado = {
    'Inscricao': '123',
    'CNPJ': '48797752000151',
    'RazaoSocial': 'Produtora xyz'
  };
  expect(logSorteio(123, sorteado)).toMatchSnapshot();
  expect(logPareceristaAdicionado()).toMatchSnapshot();
  expect(logPareceristaNaoAdicionado()).toMatchSnapshot();
});
