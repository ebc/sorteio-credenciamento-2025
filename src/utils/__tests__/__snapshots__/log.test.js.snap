// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loga corretamente parametros sorteio 1`] = `
"Sorteio realizado em: 11/11/2018 11:11:11.

Lista de habilitados possui 1 itens 
1	123	48797752000151	Produtora xyz

Semente gerada a partir do momento exato do sorteio e da lista de participantes 
semente1234

"
`;

exports[`loga corretamente sorteio 1`] = `"[Sorteado nº 124 ocupada por 123	48797752000151	Produtora xyz - "`;

exports[`loga corretamente sorteio 2`] = `
"adicionado]
"
`;

exports[`loga corretamente sorteio 3`] = `
"sorteado anteriormente, não adicionado]
"
`;
